build/release/containers.sh

```diff
@@ -75,22 +75,21 @@ ENTRYPOINT ["/usr/bin/rookd"]
 EOF
     build_artifact $os $arch $tmpdir rook/rookd rookd
     rm -fr $tmpdir
-
-    # TODO: build the client and toolbox for arm
-    [[ ${arch} == "amd64" ]] || return 0
  
     echo "Building the rook-client container"
     mkdir $tmpdir
     cat <<EOF > $tmpdir/Dockerfile
-FROM ubuntu
-RUN apt-get update && \
-    apt-get install -yq --no-install-recommends wget s3cmd kmod module-init-tools sudo
```

Comment:
it is built into the alpine image too. I brought up a fresh VM for kubernetes and ensured that everything worked in a pristine environment where the kernel module shouldn't have been already loaded before running the `mount` command